server/publications/selectedProduct.js

```diff
@@ -0,0 +1,3 @@
+Meteor.publish("selectedProduct", function(searchedCode){
+  return Products.find({itemLookUpCode: searchedCode});
```

Comment:
You could add a couple spaces in here ```{itemLookUpCode: searchedCode}``` to make it follow the meteor style guide more closely. Again, I'm just being nit picky.